algorithms/src/sorting/Kokkos_SortPublicAPI.hpp

```diff
@@ -71,20 +71,34 @@
 
 namespace Kokkos {
 
+// ---------------------------------------------------------------
+// basic overloads
+// ---------------------------------------------------------------
+
+// clang-format off
 template <class ExecutionSpace, class DataType, class... Properties>
-std::enable_if_t<(Kokkos::is_execution_space<ExecutionSpace>::value) &&
-                 (!SpaceAccessibility<
-                     HostSpace, typename Kokkos::View<DataType, Properties...>::
-                                    memory_space>::accessible)>
+std::enable_if_t<
+  (Kokkos::is_execution_space<ExecutionSpace>::value) &&
+  (!SpaceAccessibility<
+     HostSpace, typename Kokkos::View<DataType, Properties...>::memory_space
+    >::accessible)
+  >
```

Comment:
I really dislike enabling and disabling `clang-format` in different scopes.
```suggestion
std::enable_if_t<
  (Kokkos::is_execution_space_v<ExecutionSpace>) &&
  (!SpaceAccessibility<
     HostSpace, typename Kokkos::View<DataType, Properties...>::memory_space
    >::accessible)
  >
// clang-format on  
```